### DISCUSSION TITLE
Remember password for samba share

### DISCUSSION THREAD

**Discussion by gfreccia** [ltsp/ltsp#288] [Q&A]
Hi All,

I've a problem with "remember password" on Samba share.
If I check "remember password" when I create the bookmark, it works properly but only when the user is logged through the server.
When the same user is logged on Fat Client the password is always asked even if "remember password" is checked.

![Schermata del 2020-06-24 23-31-23](https://user-images.githubusercontent.com/35954627/85629861-e43dbe00-b672-11ea-9eab-ec3ec44c2c95.png)

Thank you!

**Comment by alkisg** [comment]
Hi, this is a bug in gnome-keyring-daemon, have a look at this issue: https://github.com/ltsp/ltsp/issues/22

**Comment by gfreccia** [comment]
Hi Alkis,

you're great as always!

Solved adding this on lts.conf:
_POST_INIT_FIX_PAMLTSP="sed 's/gnome-keyring-daemon/unused/g' -i /usr/share/ltsp/client/login/pamltsp"_

Thank you very much!

**Comment by Pavel202by** [comment]
Hi.
I have this problem with FAT client on ubuntu 20.04: samba passwords saved on server via x2go or in a local session, but not saved on the FAT client. I found that "/usr/bin/gnome-keyring-daemon" is not available in the ltsp image, but is available in the chroot of that image. This means that the package is not included in the build image.
I created a file "/etc/ltsp/image.excludes" with OMIT_IMAGE_EXCLUDES="/usr/bin/gnome-keyring-daemon" and rebuild image, but it didn't help.

**Comment by alkisg** [comment]
@Pavel202by please do read the related issue that I posted in my previous answer: https://github.com/ltsp/ltsp/issues/22
